### DISCUSSION TITLE
import with glob paths and jest

### DISCUSSION THREAD

**Discussion by AnsonT** [parcel-bundler/parcel#5236] [Q&A]
Is it possible to setup jest such that it can support parcel's import resolution with glob file path?
i.e. using jest to test a file with:

import foo from "./foo/*.ts"

**Comment by mischnic** [comment]
See https://github.com/parcel-bundler/parcel/discussions/5217.

You could use `moduleNameMapper`  to remap imports to `*.[extension]` to some dummy file instead.
